lib/chef/application/windows_service.rb

```diff
@@ -189,7 +189,11 @@ def run_chef_client
           config_params += " -c #{Chef::Config[:config_file]}" unless  Chef::Config[:config_file].nil?
           config_params += " -L #{Chef::Config[:log_location]}" unless Chef::Config[:log_location] == STDOUT
           # Starts a new process and waits till the process exits
-          result = shell_out("chef-client #{config_params}", :timeout => Chef::Config[:windows_service][:watchdog_timeout])
+          result = shell_out(
+            "chef-client #{config_params}",
+            :timeout => Chef::Config[:windows_service][:watchdog_timeout],
+            :logger => Chef::Log
```

Comment:
`Chef::Log` is what I tested with and worked. Just looked at mixlib-log again and looks like `logger` returns the internal logging instance which should also work. calling the various `warn`, `debug`, etc calls directly to `Chef::Log` sends those to logger. So either `Log` or `.logger` appear to be effectively the same.